src/utils.js

```diff
@@ -306,6 +306,13 @@ const getSubscriptionContactText = (contactEmail) => {
   return `${contactText}.`;
 };
 
+function truncateString(str, maxStrLength = 10) {
```

Comment:
Not for right now, but this seems like the type of util we might want to add to https://edx.github.io/frontend-platform/module-Utilities.html